include/modem/location.h

```diff
@@ -86,11 +86,14 @@ enum location_event_id {
 
 /** Result of the external request. */
 enum location_ext_result {
-	/* Location result is successful. */
+	/** Location result is successful. */
```

Comment:
```suggestion
	/** Location request was successful. */
```

Reply:
Hmmm, this might be a bit confusing as this is not used to determine the result of the `location_request()` so I remember I've tried to avoid using "location request" wording to mean anything else than the actual `location_request()` function call from the application to the Location library.

Reply:
Well, the comment for the enum says "Result of the external request". Maybe "external request" is the term which should be used here?

Reply:
I've modified these to use "cloud location request" terminology. That's where it's used. Please check the wording.